ush/sample_config/ARL_run/config.yaml_cold_aqmna13km_1day_wcoss2

```diff
@@ -63,7 +63,7 @@ cpl_aqm_parm:
   AQM_CONFIG_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/aqm/epa/data
   AQM_BIO_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/aqm/bio
   AQM_BIO_FILE: BEIS_RRFScmaq_C775.ncf
-  DO_AQM_FENGSHA: false
+  DO_AQM_FENGSHA: true
   AQM_FENGSHA_DIR: /lfs/h2/emc/ptmp/barry.baker
```

Comment:
@bbakernoaa, can you replace all `FENGSHA`s with `DUST` in the scripts and yaml files?

Reply:
did that do it all?